### ISSUE TITLE
Meetup GraphQL Migration (by Winter 2021)

### ISSUE THREAD

**Issue by Sean12697** [inevitable-team/compiled-mcr-events#103]
The following integration / Endpoints will need to be modified to use the new GraphQL API, returning data as we are currently expecting it. This can be achieved by following the [migration instructions](https://www.meetup.com/api/migrating).

https://github.com/inevitable-team/compiled-mcr-events/blob/09500872b77c42610594dda99d5b5499799eacba/_data/sources/meetup.js#L8-L10

**Comment by Sean12697** [comment]
OAuth Client is Pending.

**Comment by Sean12697** [comment]
![image](https://user-images.githubusercontent.com/22508776/135870834-91b6dfb0-ae14-43b0-8884-3e7da0232384.png)

**Comment by prototype99** [comment]
#108 may be a fix if testing is successful

**Comment by Sean12697** [comment]
OAuth Client is *still* Pending.

**Comment by Sean12697** [comment]
I've been able to get an OAuth Client approved, whipped up an [ExpressJS](https://expressjs.com/) app using the [Passport.js OAuth2 Strategy](http://www.passportjs.org/packages/passport-oauth2/), deployed it to [Heroku](https://heroku.com/), gained an `access_token` as specified in the Meetup [documentation](https://www.meetup.com/api/authentication/#p02-server-flow-section), did a lot of logging using `console.log()` and API testing using [Postman](https://www.postman.com/) ... Then I removed the Bearer Token ... It still worked ... Gdi 🙃

```bash
curl --location --request POST 'https://api.meetup.com/gql' \
--header 'Content-Type: application/json' \
--data-raw '{"query":"query($urlname: String!) {\n    groupByUrlname(urlname: $urlname) {\n      name,\n      link,\n      description\n    }\n}","variables":{"urlname":"Tech-for-Good-Live"}}'
```

**Comment by prototype99** [comment]
it is proving difficult to acquire a group image. research is being done to find an example of acquiring the image. 8 results have been found.

**Comment by prototype99** [comment]
other sources in case needed at a later date:

- https://github.com/hackyguru/megameetups
- https://github.com/nivaska/meetup-notifier
- https://github.com/saulgdmn/itevents-scrapy/tree/master/itevents
- https://github.com/gisyaliny/Social-Events-Script
- https://github.com/latinonetonline/LatinoNetOnline.Backend
- https://github.com/ArpanDam/Meetup_crawler

**Comment by Sean12697** [comment]
Solved by #108
